src/main/java/service/BinarySerializer.java

```diff
@@ -0,0 +1,53 @@
+package service;
+
+import data.interfaces.IGame;
+import service.interfaces.IFileHandler;
+
+import java.io.*;
+
+public class BinarySerializer implements IFileHandler {
+
+    @Override
+    public IGame readGameFile(String path) {
+        IGame game;
+        File input = new File(path);
+
+        if(!input.exists()) {
+            return null;
+        }
+
+        try {
+            FileInputStream fis = new FileInputStream(input);
+            ObjectInputStream ois = new ObjectInputStream(fis);
+            game = (IGame) ois.readObject();
```

Comment:
sollten die nicht auch wieder geschlossen werden?
```suggestion
            ois.close()
```

Reply:
Vielleicht am besten auch in einem finally Block